maven-surefire-common/src/main/java/org/apache/maven/plugin/surefire/AbstractSurefireMojo.java

```diff
@@ -4215,4 +4213,14 @@ private Classpath setCachedClasspath( @Nonnull String key, @Nonnull Set<Artifact
             return classpath;
         }
     }
+
+    /**
+     * No tests run failures.
+     */
+    enum NoTestsFailureReason
+    {
+        NO_FAILURE,
+        NO_SPECIFIED_TESTS_RUN,
```

Comment:
... SPECIFIED_TESTS_RUN here...

Reply:
Pls check the names of enum now, see the [commit](https://github.com/apache/maven-surefire/pull/350/commits/3e8c22489648e06a520771c5b857088bad0e565b).